target_chains/ethereum/contracts/forge-test/EntropyAuthorized.t.sol

```diff
@@ -174,4 +174,138 @@ contract EntropyAuthorized is Test, EntropyTestUtils {
         vm.expectRevert(EntropyErrors.Unauthorized.selector);
         random.acceptAdmin();
     }
+
+    function testWithdrawFeeByAdmin() public {
+        // Register provider1 first
+        bytes32[] memory hashChain = generateHashChain(provider1, 0, 100);
```

Comment:
there's a lot of code duplication in these tests. I suggest adding a helper method that sets up the contract with fees so the test simply has to do the withdraw call.